Statistics/Statistics.cs

```diff
@@ -1,12 +1,38 @@
 ﻿using System;
 using System.Collections.Generic;
+using System.Linq;
 
 namespace Statistics
 {
-    public class StatsComputer
+    public class Stats
     {
-        public Stats CalculateStatistics(List<float> numbers) {
-            //Implement statistics here
+        public float average;
+        public float max;
+        public float min;
+        public Stats()
+        {
+            this.average = float.NaN;
+            this.max = float.NaN;
+            this.min = float.NaN;
         }
     }
+    public class StatsComputer
+    {  
+        public Stats CalculateStatistics(List<float> numbers) {
+            
+            //Implement statistics here
+            Stats stats = new Stats();
+            if (numbers.Count!= 0)
+            {
+                if (numbers.Contains(float.NaN))
+                {
+                    numbers.RemoveAll(float.IsNaN);
```

Comment:
Nice and clean filter for NaN. However, beware of side-effects. Check if this changes the list for the caller as well? When callers call the function 'CalculateStatistics', they don't expect the list to be changed inside the function. E.g., if they want to count the number of error measurements (NaNs) after calculating statistics.
Write a test to protect against the list being modified for callers and then take the next step.